core/services/ocr2/delegate.go

```diff
@@ -207,6 +206,36 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 	var pluginOracle plugins.OraclePlugin
 	var ocr2Provider types.Plugin
 	switch spec.PluginType {
+	case job.Mercury:
+		mercuryProvider, err2 := relayer.NewMercuryProvider(
+			types.RelayArgs{
+				ExternalJobID: jb.ExternalJobID,
+				JobID:         spec.ID,
+				ContractID:    spec.ContractID,
+				New:           d.isNewlyCreatedJob,
+				RelayConfig:   spec.RelayConfig.Bytes(),
+			}, types.PluginArgs{
+				TransmitterID: spec.TransmitterID.String,
+				PluginConfig:  spec.PluginConfig.Bytes(),
+			})
+		if err2 != nil {
+			return nil, err2
+		}
+		oracleArgsNoPlugin := libocr2.OracleArgs{
+			BinaryNetworkEndpointFactory: peerWrapper.Peer2,
+			V2Bootstrappers:              bootstrapPeers,
+			ContractTransmitter:          mercuryProvider.ContractTransmitter(),
+			ContractConfigTracker:        mercuryProvider.ContractConfigTracker(),
+			Database:                     ocrDB,
+			LocalConfig:                  lc,
+			Logger:                       ocrLogger,
+			MonitoringEndpoint:           d.monitoringEndpointGen.GenMonitoringEndpoint(spec.ContractID, synchronization.OCR2Median),
```

Comment:
telemetry types were [recently added](https://github.com/smartcontractkit/chainlink/pull/8318) to help with routing in the ingress service. could we add a new type for mercury and use that instead of `OCR2Median`?

https://github.com/smartcontractkit/chainlink/blob/develop/core/services/synchronization/common.go#L6-L12